sources/packages/web/src/components/common/InstitutionUserSummary.vue

```diff
@@ -1,57 +1,55 @@
 <template>
   <!-- This component is shared between ministry and student users -->
-  <div class="row p-3">
-    <div class="col col-3">
-      <span class="category-header-large color-blue">
-        All Users({{ usersListAndCount.totalUsers }})
-      </span>
-    </div>
-    <div class="col">
-      <div class="row float-right">
-        <InputText
-          v-model="searchBox"
-          placeholder="Search User"
-          @keyup.enter="searchUserTable()"
-        />
-        <v-btn
-          @click="searchUserTable()"
-          tile
-          class="ml-2 primary-btn-background"
-        >
-          <font-awesome-icon :icon="['fas', 'search']" />
-        </v-btn>
+  <div class="mb-4">
```

Comment:
IMO this component is not a good one to be shared because the Ministry one is going to be way too simple and the Institution will be more complex.
This could be noticed in the amount of times that there is a "if" to check the client type consuming it. Even the API call are different.
Just would like to share these thoughts, if there is a Team decision to move on as it is it is fine 😉